core/src/com/etheller/warsmash/datasources/CascDataSourceDescriptor.java

```diff
@@ -84,13 +84,8 @@ else if (!this.gameInstallPath.equals(other.gameInstallPath)) {
 			return false;
 		}
 		if (this.prefixes == null) {
-			if (other.prefixes != null) {
```

Comment:
This method was computer generated by my IDE. If we change it, then I generate another similar method later, they will become needlessly different. What do we propose as a good solution to this dilemma? Ideally, all the computer generated methods will end up the same.

Reply:
I was not aware that it was generated. How and where can I find the information about generated code parts to avoid making changes on them? Meanwhile, will revert this change.

Reply:
Reverted in 6612bf8bc79b5782bcbc80a75a2a73567289cf87.